axelor-stock/src/main/resources/views/StockMove.xml

```diff
@@ -210,7 +210,9 @@
                     <viewer>{{record.toAddressStr}}</viewer>
                 </field>
             </panel>
-
+            
+            <field name="$filterOnAvailableProducts" title="Filter on Available Products" type="boolean" onChange="save,action-stock-move-method-update-stock-move-line"/>
```

Comment:
Add a real field on stockMove instead of dummy.  Don't save on onChange, but just update field on all lines.